src/io/yaml.cpp

```diff
@@ -0,0 +1,27 @@
+#include "aikido/io/yaml.hpp"
+
+namespace aikido {
+namespace io {
+
+YAML::Node loadYAML(
+    const dart::common::Uri& yamlUri,
+    const dart::common::ResourceRetrieverPtr& retriever)
+{
+  // These can be replaced with retriever::readAll(uri) with Dart 6.2.
+  // std::string content = retriever->readAll(yamlUri);
```

Comment:
Yeah, let's use `readAll()` as AIKIDO now depends on DART 6.3. :tada:

Reply:
I think `CMakeLists.txt` might not reflect this dependency? https://github.com/personalrobotics/aikido/blob/master/CMakeLists.txt#L129

Reply:
We should (1) update `CMakeLists.txt` to reflect the DART 6.3 dependency and (2) replace all of this code with the line above.